.github/workflows/rust.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     - name: Install MSRV
       uses: actions-rs/toolchain@v1
       with:
-        toolchain: 1.65.0
+        toolchain: 1.70.0
```

Comment:
is this bump caused by the cli::clap bump ?
As far as i'm concerned, i'm not bothered by complying to very recent compilers.
But if the library differs too much from the command line app/demo, it could be interesting that we differentiate their requirements to end users

Reply:
Yes indeed. That version of `clap` comes with a requirement on `1.70`. In general I also prefer using recent compilers / pacakges if there is no other restriction. 

One idea could be to add the following `rust-version = "1.70"` in the Cargo.toml manifest of `ublox_cli`.

Reply:
Yep, we need to do just that

Reply:
Will do later today.

Reply:
I moved the example to an `examples` folder and excluded it from the root workspace